web/src/engine/StorageController.ts

```diff
@@ -46,6 +46,7 @@ export function SanitizeFileName(name: string): string {
         .replace(patternControlCharsUTF8, '')
         .replace(/./g, c => lookup[c] ?? c)
         .replace(/[\s.]+$/, '')
+        .replace(/~$/, '')//While Windows allows it, it seems NodeJS somehow dont like paths with hyphen at the end
```

Comment:
Comment is incorrect, NodeJS can handle trailing tilde, the Browser's [File System API](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemDirectoryHandle) cannot.
Use a more appropriate replacement (e.g., [～](https://unicode-explorer.com/c/FF5E))
➔ Probably best to replace all tilde characters with a unicode variant to keep consistence.

Reply:
Aaaaah thats the reason, thanks !